scripts/build-crossdock.sh

```diff
@@ -17,7 +17,7 @@ if [[ "$BRANCH" == "main" ]]; then
   docker buildx build --push \
     --progress=plain \
     --platform=linux/amd64 \
-    ${IMAGE_TAGS} \
+    "${IMAGE_TAGS[@]}" \
```

Comment:
IMAGE_TAGS needs to be assembled as array in order to use `[@]` syntax. Currently it is assembled as a string. This change is the same as `"${IMAGE_TAGS}"`:
```shell
$ cat 1.sh
function pr {
  echo "arg1=$1"
  echo "arg2=$2"
  echo "arg3=$3"
  echo "arg4=$4"
}

x="x y z"
x="${x} w"
pr "${x[@]}"
```
```shell
$ bash 1.sh
arg1=x y z w
arg2=
arg3=
arg4=

Reply:
does this work?
```bash
x=("x" "y" "z")
x+=("w")
for z in "${x[@]}"; do
  echo "x=$z"
done 
```
```
output
x=x
x=y
x=z
x=w
```

Reply:
yes